circleguard/utils.py

```diff
@@ -220,6 +220,43 @@ def hitradius(CS):
     return np.float32(64 * ((1.0 - np.float32(0.7) * (float(np.float32(CS)) - 5) / 5)) / 2) * np.float32(1.00041)
 
 
+def get_quartiles(arr):
+    """
+    Getting the quartiles in order to use Tukeys method to eliminate the outliers in hits
+
+    This function only gets called by 'EliminateOutliers'
+    """
+    arr = np.sort(arr)
+    mid = len(arr)//2
+    if(len(arr)%2 == 0):
+        q1 = np.median(arr[:mid])
+        q3 = np.median(arr[mid:])
+    else:
+        q1 = np.median(arr[:mid])
+        q3 = np.median(arr[mid+1:])
+    return q1,q3
+    
+def eliminate_outliers(arr, bias = 1.5):
+    """
+    Starting function to eliminate outliers in hits.
+    This function gets rid of super late or early hits in order to represent UR a bit better 
+    in order to determine on how spread the majority of hits are   
+
+    Parameters
+    ----------
+    arr: list:`numeric list`
+        List of hits
+    bias = numeric:`number`
+        How high the bias should be (standard is 1.5)
+
+    """
+    q1,q3 = get_quartiles(arr)
```

Comment:
instead of writing our own `get_quartiles` function, I would use `np.percentile(x, [75 ,25])` to get the 75th and 25th percentile (q3 and q1 respectively I believe?): https://stackoverflow.com/a/23229224/12164878